tools/tests/test_nd_freeze

```diff
@@ -198,4 +198,12 @@ test_setup "neurodebian" "jessie" " " "-h"
 assert_line_in_file "stdout" "Usage: nd_freeze"
 test_teardown
 
+echo "[ Test unsigned repos ]"
+test_setup "neurodebian" "sid" "20140828" ""
+assert_line_in_file "stdout" "W: GPG error: http://snapshot-neuro.debian.net/archive/neurodebian/20140828T050524Z sid InRelease: The following signatures were invalid: DD95CC430502E37EF840ACEEA5D32F012649A5A9"
+assert_line_in_file "stdout" "W: GPG error: http://snapshot-neuro.debian.net/archive/neurodebian/20140828T050524Z data InRelease: The following signatures were invalid: DD95CC430502E37EF840ACEEA5D32F012649A5A9"
+assert_line_in_file "stdout" "INFO: Discovering installed packages for possible version updates"
+assert_line_in_file "stdout" "INFO: UPDATING:"
+test_teardown
```

Comment:
Does this assure that the downgrades worked out correctly?  I see only the checks for seeing messages stating the intent (UPDATING) and errors, but not that it all succeeded, or that is assumed and `test_setup` would otherwise fail?

Reply:
Was this one addressed?

Reply:
Yes, checks were added for the downgrades